packages/shared/src/urbit/utils.ts

```diff
@@ -424,7 +424,7 @@ export function extractGroupPrivacy(
 }
 
 export function createSectionId() {
-  const idParts = formatUv(BigInt(Date.now())).split('.');
```

Comment:
this should work fine hmm. what did you see as the error?

Reply:
[ManageChannelsScreenView] Failed to create section: TypeError: x.neq is not a function
    at formatUv (uv.ts:24:12)
    at createSectionId (utils.ts:427:19)
    at Module.addNavSection (groupActions.ts:336:24)
    at useGroupContext.ts:136:21
    at onSave (ManageChannelsScreenView.tsx:757:19)
    at EditSectionNameSheet.tsx:29:7
    at createFormControl.ts:1153:17

Reply:
https://github.com/urbit/aura-js/blob/master/src/uv.ts I think our module is out of date somehow because that's not even present anymore

Reply:
I think tlon-mobile and tlon-web-new have it listed as 1.0.0

Reply:
Maybe we didn't release a new version of aura after the bigint changes?

Reply:
Or we just didn't bump it in our apps